### DISCUSSION TITLE
folders view

### DISCUSSION THREAD

**Discussion by macst34** [jellyfin/jellyfin-androidtv#2666] [General]
Hello all, 

I was wondering if the androidtv jellyfin client will implement the 'folders' view ?

on the server is an option:

![image](https://user-images.githubusercontent.com/5762387/232006283-53ff3b6b-376a-4f45-a7ba-f50564331e8f.png)

and on the web client looks like this:  

![image](https://user-images.githubusercontent.com/5762387/232007872-4762b7c8-f514-40db-b737-edc9a7916e80.png)

wondering if this is/will be implemented in the androidtv jellyfin client.

Thanks on all the great work!

-m

**Comment by nielsvanvelzen** [comment]
The app does not support the folder view right now. It will likely be added when the user inteface is rewritten as it's quite the challenge to add in the current codebase.

**Reply by Dikakus** [reply]
That would  be a great feature. I have all my music organized by folders as the maintenance of the metadata music is quite expensive, as you know. :D

**Comment by lorient7** [comment]
I have also noticed that this functionality is still not implemented in the Android TV client.

It would be very useful if the folder view could be supported, in the same way it is available in the web interface and on other clients. Many users organize their media libraries manually using folders, and we would like to be able to access our multimedia content in a structured and organized way according to our own preferences.

Without this feature, the Android TV client forces all content to be displayed inside broad containers such as Movies or Series, which can feel like having everything mixed together in one big unordered bag. This makes browsing more difficult, especially for users with large libraries or custom folder structures.

Having folder view support in the Android TV app would avoid the need to use other applications, such as Kodi, just to navigate media in an organized way.

Thank you for the great work on Jellyfin. I hope this feature can be considered for a future version of the Android TV client.

**Comment by AlexD81** [comment]
+1  for folder view
